ansible/inventories/sandbox/group_vars/catalog-web-next/vars.yml

```diff
@@ -3,6 +3,22 @@
 # https://github.com/GSA/datagov-infrastructure-modules/issues/32
 ansible_python_interpreter: /usr/bin/python3
 
-postgresql_login_host: "{{ catalog_next_postgresql_login_host }}"
-postgresql_login_password: "{{ catalog_next_postgresql_login_password }}"
-postgresql_login_user: "{{ catalog_next_postgresql_login_user }}"
+catalog_next: true
+uses_gunicorn: true
+
+catalog_ckan_service_url: https://catalog-next.sandbox.datagov.us
+ckan_site_domain: "{{ urls_public.catalog_next }}"
+catalog_ckan_app_version: master
+app_repo: https://github.com/GSA/catalog.data.gov
+
+# postgresql_login_host: "{{ catalog_next_postgresql_login_host }}"
+# postgresql_login_password: "{{ catalog_next_postgresql_login_password }}"
+# postgresql_login_user: "{{ catalog_next_postgresql_login_user }}"
+
+catalog_ckan_db_host: "{{ catalog_next_postgresql_login_host }}"
+catalog_ckan_db_name: catalog_db_next
+catalog_ckan_db_pass: "{{ catalog_next_postgresql_login_password }}"
+catalog_ckan_db_user: "{{ catalog_next_postgresql_login_user }}"
+
+catalog_ckan_plugins_default: []
```

Comment:
Let's get the theme extension in there https://github.com/GSA/catalog.data.gov/blob/master/.env#L43

Reply:
Yep!  Will do.  When I started this there was no theme :-)  Although it is proving a little challenging to generate a usable frozen requirements file.